CONTRIBUTING.md

```diff
@@ -3,6 +3,14 @@
 If you would like to become an active contributor to this project please
 follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](https://opensource.microsoft.com/collaborate/).
 
+## Pre-requisites
+
+Install pre-reqs:
+
+```bash
+pip install azure_devtools pytest pytest-azurepipelines pytest-cov
+```
+
```

Comment:
Would you think we could add `pytest-azurepipelines` and `pytest-cov` into `requirements-dev.txt` file so that these packages get installed automatically when we create the `azurequantum` Conda environment?
```bash
conda env create -f environment.yml
```
We already seem to install dev packages [this way](https://github.com/microsoft/qdk-python/blob/68a3fd28e1f62c0d098afa15dd03ad08d13d006a/azure-quantum/requirements-dev.txt#L2C1-L2C27)

Reply:
That makes sense! Can we do that changes as a separate PR in case it causes breakages in pipelines?

Reply:
Discussed offline. Will consider adding the dependencies into `requirements-dev.txt` in a separate PR